gcp_variant_transforms/testing/integration/run_tests.py

```diff
@@ -188,22 +187,31 @@ def run_assertion(self):
 class QueryFormatter(object):
   """Formats a query.
 
-  Replaces keyword TABLE_NAME and eventually macros in the query.
+  Replaces keyword TABLE_NAME and macros in the query.
   """
 
+  NUM_ROWS = 'SELECT COUNT(0) AS num_rows FROM {TABLE_NAME}'
```

Comment:
nit: to make these distinct from the rest of the variables (e.g. TABLE_NAME), what do you think of adding a `_QUERY` suffix to all of them?

Reply:
Done.